### ISSUE TITLE
Unhandled event error thrown

### ISSUE THREAD

**Issue by paul-barry-kenzan** [kenzanlabs/generator-keystone#22]
After relocating specific `devDependencies` to the `dependencies` property ( #19 ), and globally installing the latest `yo` package ( #21 ), I recieved the following error when the generator attempts to copy template files.

```bash
events.js:154
      throw er; // Unhandled 'error' event
      ^
AssertionError: Trying to copy from a source that does not exist: /usr/local/lib/node_modules/generator-keystone/app/templates/.gitignore
    at EditionInterface.exports._copySingle (/usr/local/lib/node_modules/generator-keystone/node_modules/mem-fs-editor/lib/actions/copy.js:45:3)
    at EditionInterface.exports.copy (/usr/local/lib/node_modules/generator-keystone/node_modules/mem-fs-editor/lib/actions/copy.js:23:17)
    at writing (/usr/local/lib/node_modules/generator-keystone/app/index.js:310:13)
    at Object.<anonymous> (/usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/lib/index.js:392:23)
    at /usr/local/lib/node_modules/generator-keystone/node_modules/run-async/index.js:25:25
    at /usr/local/lib/node_modules/generator-keystone/node_modules/run-async/index.js:24:19
    at /usr/local/lib/node_modules/generator-keystone/node_modules/yeoman-generator/lib/index.js:393:9
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

With some research, it seems it's caused by npm. If npm doesn't find a `.npmignore` file, it will rename the `.gitignore` file to be ".npmignore", thus causing our copy script to fail. 

Relevant Issues:
- https://github.com/yeoman/generator-polymer/issues/240
- https://github.com/PolymerElements/seed-element/pull/69
- https://github.com/npm/npm/issues/1862

A potential solution would be to A) rename our `.gitignore` template to `_gitignore`; or B) add a `.npmignore` file to our project.

I feel (A) is the easier solution in the current setup, but will try (B) if (A) doesn't resolve the issue.

**Comment by thescientist13** [comment]
seems this can be closed since #25  was fixed?  (I think Github doens't close issues when there are links to other open issues)

**Comment by paul-barry-kenzan** [comment]
Ahh, that really stinks that GH won't close auto-magically. 

Closing as resolved with #25
